public/app/api/clients/folder/v1beta1/hooks.ts

```diff
@@ -252,3 +288,28 @@ function getUserKeys(resultFolder: ReturnType<typeof useGetFolderQuery>): string
       ].filter((v) => v !== undefined)
     : [];
 }
+
+const appPlatformFolderToLegacyFolder = (
+  folder: Folder
+): Omit<FolderDTO, 'parents' | 'canSave' | 'canEdit' | 'canAdmin' | 'canDelete' | 'createdBy' | 'updatedBy'> => {
+  // Omits properties that we can't easily get solely from the app platform response
+  // In some cases, these properties aren't used on the response of the hook,
+  // so it's best to discourage from using them anyway
+
+  const { annotations, name, creationTimestamp, generation, labels } = folder.metadata;
+  const { title } = folder.spec;
+  return {
+    id: parseInt(labels?.[DeprecatedInternalId] || '0', 10) || 0,
+    uid: name!,
+    title,
+    // general folder does not come with url
+    // see https://github.com/grafana/grafana/blob/8a05378ef3ae5545c6f7429eae5c174d3c0edbfe/pkg/services/folder/folderimpl/folder_unifiedstorage.go#L88
+    url: name === GENERAL_FOLDER_UID ? '' : getFolderUrl(name!, title!),
+    created: creationTimestamp || '0001-01-01T00:00:00Z',
+    updated: annotations?.[AnnoKeyUpdatedTimestamp] || '0001-01-01T00:00:00Z',
+    managedBy: annotations?.[AnnoKeyManagerKind] as ManagerKind,
```

Comment:
There's no way around this currently and we're not planning to fix this, should we add an eslint-ignore here?

Reply:
Yup, I had already actually fixed it in `main`, have fixed it now when fixing conflicts :+1: